index.js

```diff
@@ -1,33 +1,31 @@
-// Without "./", Node looks
-// in its global module folder
+//without "./", node looks in it's global module folder
+
 const http = require("http");
-function handleRequestAndResponse(req, res) {
-    console.log(req.url);
-    const timestamp = new Date();
-    console.log("Received a Request");
-    res.end(`<h1>Hello World ${timestamp}</h1>`);  // .end() sends the response.
-                                          // It "tapes up the box and ships it".
-}
-// What's a server??!?
-// A server is a piece of code
-// or a computer running a piece of code
-// that does something useful for YOU.
-// Usually, servers store
-// data and often manipulate that data.
-// const server = http.createServer(handleRequestAndResponse);
-const server = http.createServer((req, res) => {
+const nerds = require('nerds');
+const single = nerds.resolve('Harry Potter').asArray()[0].full;
+console.log(single);
+
+
+function handleReqAndResp(req, res) {
     console.log(req.url);
-    const timestamp = new Date();
-    console.log("I totally received a Request!!!!!!!!!!!!!!!!!!!");
-    res.end(`<h1>Hello World ${timestamp}</h1>`);  // .end() sends the response.
-                                          // It "tapes up the box and ships it".
-});
+    res.end(`<h1>Hello World this is ${single}</h1>`); //.end() sends the response.
```

Comment:
Out of curiosity, do you have to restart the program to get a different Harry Potter character name? If you wanted to get a different name on every refresh of the browser, what would you change?